### ISSUE TITLE
Create Java API

### ISSUE THREAD

**Issue by sramsay** [CDRH/abbot#2]
Abbot should have a Java API that looks more-or-less like:

```
import edu.unl.abbot.Abbot;

    public class APITest {

    public static void main(String[] args) {

        Abbot abbot = new Abbot();

        abbot.convert("/home/sramsay/projects/abbot/input", "/home/sramsay/projects/abbot/output/");

    }

  }
```

**Comment by sramsay** [comment]
Completed (in API branch).  Will close when Issue 4 (External files) is resolved and branch is merged.

**Comment by smutniak** [comment]
I'm presuming that all input data files will have a .xml extension.  Is that true?
What extension will the conversion specification file have?
thanks

**Comment by sramsay** [comment]
Resolved for 0.3.3.  Please add new issues for problems related to the API.
